### ISSUE TITLE
inplace argument deprecated

### ISSUE THREAD

**Issue by mschrimpf** [brain-score/brainio_base#3]
https://github.com/brain-score/brainio_base/blob/4d733ce7cede06195ec13b6fae1678178bcff356/brainio_base/assemblies.py#L248 
and 
https://github.com/brain-score/brainio_base/blob/4d733ce7cede06195ec13b6fae1678178bcff356/brainio_base/assemblies.py#L213
yield
```
FutureWarning: The inplace argument has been deprecated and will be removed in xarray 0.12.0.
  result.set_index(append=True, inplace=True, **{self.multi_group_name: list(self.group_coord_names)})
```

On pip, xarray is already past version 0.12 so we need to fix this.
I'm not sure how to work around it, since the method in question is setting an index on `self`.

**Comment by mschrimpf** [comment]
Two possible options we thought of so far:
1. use `def`s and under-scored classes
```
class _DataAssembly(DataArray):
    ...

def DataAssembly(*args, **kwargs):
    a = _DataAssembly(*args, **kwargs)
    a = gather_indexes(a)
    return a
```

2. add class.create method
```
class DataAssembly(DataArray):
    @classmethod
    def create(*args, **kwargs):
        a = DataAssembly(*args, **kwargs)
        a = gather_indexes(a)
        return a
````

For now though, we keep xarray at version 0.12 (#5)

**Comment by jjpr-mit** [comment]
Fixed by #14
